hack/mkdocs-generate-conformance.py

```diff
@@ -50,28 +50,49 @@ def on_pre_build(config, **kwargs):
                                                 'HTTPRoutePathRedirect', 'HTTPRouteHostRewrite', 'HTTPRoutePathRewrite', 'HTTPRouteRequestMirror', 'HTTPRouteRequestMultipleMirrors', 'HTTPRouteRequestTimeout', 'HTTPRouteBackendTimeout', 'HTTPRouteParentRefPort']
 
 
+# NOTE add past versions of implementations to here when new release is cut
+past_versions = """
+??? info
```

Comment:
Recommend leaving this expanded and adding a meaningful title:
```suggestion
???+ Previous Gateway API Versions
```